src/components/Nav.tsx

```diff
@@ -15,27 +14,32 @@ const StyledNav = styled.div`
   z-index: 10;
 `;
 
-const StyledH1 = styled.h1`
+const StyledLink = styled.a.attrs(props => {
+  href: props.href
+})`
   margin-top: 24px;
   margin-bottom: 24px;
   font-size: 14pt;
+  font-weight: 700;
+  font-family: 'Roboto Slab',sans-serif;
+  text-transform: uppercase;
 `;
 
 export default function Nav() {
   return (
     <StyledNav>
-      <Link to="/">
```

Comment:
These changes are not necessary, the Gatsby `Link` component compile to HTML `a` tags with event listeners for loading pages.